### DISCUSSION TITLE
Sandpack package and Fast Refresh

### DISCUSSION THREAD

**Discussion by NinoMaj** [codesandbox/codesandbox-client#4498] [General] [answered]
When using `sandpack` standalone-package with `create-react-app` template should Fast Refresh work out of the box when using newest version of `react`, `react-dom` and `react-scripts` or that is something that end user should setup?

**Comment by CompuIves** [comment, marked as answer]
Hey! It should work out of the box, this is some functionality that isn't dependent on de host.

**Reply by NinoMaj** [reply]
After I've upgraded to v0.0.63 it started to work. Thanks @CompuIves!
